main.py

```diff
@@ -75,6 +87,21 @@ def main():
                 player_has_card = True
                 print(f"{player.name} discarded {card_on_table} (player left with {len(player.cards)} cards) \n")
                 break
+        if not player_has_card:
```

Comment:
Bug : In case the first play drop changes color (color = "Change color" , num = 0) and the second player has change color too, he will drop it too. the next if will is true
`if card.color == card_on_table.color or card.num == card_on_table.num:`